MoltenVK/MoltenVK/GPUObjects/MVKPipeline.h

```diff
@@ -99,13 +99,18 @@ class MVKPipeline : public MVKBaseDeviceObject {
 	/** Returns the current auxiliary buffer bindings. */
 	const MVKShaderAuxBufferBinding& getAuxBufferIndex() { return _auxBufferIndex; }
 
+	/** Returns whether or not full image view swizzling is enabled for this pipeline. */
+	bool fullImageViewSwizzle() const { return _fullImageViewSwizzle; }
```

Comment:
From what I can see...the value of this iVar and property never changes...and exists only to remember the state of `_device->_pMVKConfig->fullImageViewSwizzle` when the pipeline was created. Is that correct?

Reply:
That's correct.